### ISSUE TITLE
Issue with orbi pro login

### ISSUE THREAD

**Issue by stofferr** [gruijter/com.gruijter.netgear#52]
========== STARTING TEST ==========
Node version: v12.16.1
Netgear package version: 4.3.1
OS: linux 4.14.15-g3c513a6eca
Info: Homey fw:6.1.0 app: 3.2.4
t = 0
trying to auto discover Netgear routers...
{}
{host:192.168.1.1,port:80,tls:false,username:admin,password:*****,timeout:18000,sessionId:A7D88AE69687E58D9A00,loggedIn:false,configStarted:false,soapVersion:3,loginMethod:2,guestWifiMethod:{},lastResponse:{},queue:{queue:{},overheat:1620332051018,pending:false,params:{default:{rule:common,key:common},rules:{common:{rate:3,limit:1,priority:1}},overall:{rate:10,limit:1},retryTime:2,ignoreOverallOverheat:true},heatPart:100}}

I constantly have a 401 message.

**Comment by gruijter** [comment]
Do you still have the issue after giving both router and Homey a cold start (power down)?

**Comment by stofferr** [comment]
Hi Robin,
Yes I still have the HTTP 401 even after a hard power down.
Best,
Roger

**Comment by gruijter** [comment]
Did you also try to manually fill in the IP and port (80 or 5000) when running the test?

And is the official orbi app able to login and show the attached devices?

**Comment by stofferr** [comment]
Hi Robin,

I have not used the ORBI app itself because it keeps insisting I start
initial configuration of my orbi setup. I am not comfortable starting this
process as I fear all my settings are reset to defaults.

What I CAN tell is this:
- I can reach the SRR60 using the local web browser and username password
as also used in Homey.

- I have tested the following login settings:
    - choose auto logon - fail 401
    - choose new login - fail 401
    - choose old login - fail 401

- I have also used the 'test functionality' in the settings. There the
system discovers another incorrect IP address. This obviously fails.
Changing the IP address manually DOES work and all login methods succeed.
Many details are retrieved using the test functionality after
manually fixing the IP address.

- When removing the device from Homey and adding it again, enter all
settings correct, then the: model, firmware version, device mode and serial
number are correctly retrieved. (verified in settings screen of the device).

However after a little while (maybe 5-7 seconds) the device main page
mentioned HTTP 401 again.

**Comment by gruijter** [comment]
Can you send me app diagnostics report after repeating what you just described? 

So first run a test with manual ip
Then add the device with manual ip
Let it run until you get the error

Then make a diagnostics report from the app settings.

**Comment by stofferr** [comment]
Diagnostics report: 5632e1ff-05a0-4be0-ae69-bf1bb2a5a898

**Comment by gruijter** [comment]
I see a few things from the report.

* it seems you already have a router added as device to Homey.
* I suspect you have multiple devices (satellites?) in your network. This is causing autodiscovery to fail.
* the orbi reports a soap port of 443
* the orbi is not on the latest firmware

So please do the following:

1) upgrade firmware of all units to latest version. https://kb.netgear.com/000062986/SRR60-SRS60-Firmware-Version-2-6-2-200

2) remove the homey app an reinstall it

3) go to app settings and run a compatibility test. Manually fill in the IP and set port to 443.

4) paste the result of the test here

**Comment by stofferr** [comment]
Hi, this firmware is instable. I need to wait for a more stable version
before I can continue.
Roger

**Comment by gruijter** [comment]
ok, thx for the update. Let me know once you have tried a stable release.

**Comment by stofferr** [comment]
Hi Robin,

The stability issues on my netgear ORBI (1 router and 2 satellites) are
caused by load created by your app.
After too much load is detected, my router disconnects (according to
netgear),
I have disabled the netgear app to test netgear's theory and indeed now the
internet connection is stable.
As the instability is caused by the combination of SRR60 and your Netgear
Router App, I have decided to keep the Homey App for Netgear Router
disabled.

Thanks, Roger

PS: they indicate that these calls cause significant load:
- device causing instability is sending the soap request for the
TrafficMeter and ACL functions.

**Comment by gruijter** [comment]
Lol. Yes that is one way to explain it 😂

I had the same issue with certain firmware releases on both my R7000 and R7800. Their firmware is buggy and doing many SOAP calls then makes the unit crash. The SOAP calls I am doing from Homey are exactly the same as what their own app is doing. It is just that Homey repeats that every minute.

Anyhow, you might want to give it another try when they release new firmware. Also in the Homey app you can reduce the number of calls by deselecting traffic information and set attached device method to 'old'.

But of course stable internet is much mire important then a Homey connection  😉

**Comment by stofferr** [comment]
Hi Robin,

I understand your concern.

For the time being I have followed you advice and disabled traffic/attached
device=old. I have no more instability. Now I upgraded to the latest
firmware. Still stable.
No more 401 error now, but, the statistics in the homey app (Internet
connected / #devices) are not updated either.

My setup is the following:
1 SRR60 router
2 SRS60 satellites
Connected via cable.



Best, Roger
